doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -424,11 +416,34 @@ Response code:
 * Errors use a non-200 and (optionally) an error payload. See subsection on Error Responses for more details.
 
 + Parameters
-    + limit: 20 (optional, number)
-    + offset: 20 (optional, number)
-    + type: Room (optional, string)
-    + id: Bcn_Welt (optional, string)
-    + idPattern: (optional, string)
+    + id: Boe_Idearium (optional, string) - A comma separated list of elements.
+    Retrieve entities which ID match one of the elements in the list.
+    
+    + type: Room (optional, string) -  comma separated list of elements.
```

Comment:
Related with my previous comment (https://github.com/telefonicaid/fiware-orion/pull/1218/files?diff=split#r39383034), it seems that apiary allows to specify mandataory/optional in an attribute... However, which effect has this in the "righ panel" rendering? Maybe is a subtle detail, but I cannot see how "optional" is marked in the web.